kasa/cli/main.py

```diff
@@ -208,6 +217,15 @@ def _legacy_type_to_class(_type):
     envvar="KASA_CREDENTIALS_HASH",
     help="Hashed credentials used to authenticate to the device.",
 )
+@click.option(
+    "-ex",
```

Comment:
```suggestion
```
I'd avoid doing a shortcut here, it's not something one should get accustomed to use after all :-)

Reply:
It helps me when I'm testing 😄 . I can remove if you feel strongly

Reply:
I'd rather avoid doing so, especially if we may want to ship before everything can be moved away from experimental. Better not standardize such, one can always create an alias to set the flag for dev :-)

Reply:
Ok fair enough, will drop the `-ex`